ui/src/external/monaco.flux.completions.ts

```diff
@@ -13,6 +13,83 @@ const m2p = new MonacoToProtocolConverter(),
   p2m = new ProtocolToMonacoConverter()
 
 export function registerCompletion(monaco: MonacoType, server: LSPServer) {
+  monaco.languages.registerDocumentSymbolProvider(FLUXLANGID, {
```

Comment:
does it make sense to keep all of these in the "completions" file? not arguing for a different file for each functionality change, just seems confusing that i'd look here to find code folding providers

Reply:
yeah I meant to rename that file to monaco.flux.lsp.ts instead but forgot 👍